components/admin/users/user-form.tsx

```
@@ -232,12 +238,12 @@ export default function UserForm({ user, onSuccess, isSubmitting = false, onCanc
           <div className="flex-1">
             <label className="block text-sm font-medium mb-2 text-gray-700 dark:text-gray-300">{t('AVATAR_URL')}</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)